utils/src/test/java/software/amazon/awssdk/utils/Md5UtilsTest.java

```diff
@@ -23,6 +23,7 @@
 import java.nio.charset.StandardCharsets;
 import org.apache.commons.io.FileUtils;
 import org.junit.Test;
+import software.amazon.awssdk.utils.internal.Base16;
```

Comment:
unused import?

Reply:
Whoops, forgot to push that fix. Good catch!